src/styles.scss

```diff
@@ -1,3 +1,33 @@
 @import '~ngx-toastr/toastr';
 html, body { height: 100%; }
-body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
\ No newline at end of file
+body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+
+// Button start
+  .btn {
+    width: 80px;
+    height: 30px;
+    margin: 0 15px;
+    border-radius: 10px;
+    border: none;
+    font-size: 14px;
+    box-shadow: 2px 2px 4px 0 rgba(90, 90, 90, 0.4);
```

Comment:
Might be good to define all colors in variables so there won't be any repetitions and they can be referenced in many places. Like this:

```
$base-color: #c6538c;
$border-dark: rgba($base-color, 0.88);

.alert {
  border: 1px solid $border-dark;
}
```

Reply:
fix style